client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -96,10 +95,10 @@ const Launchpad: Step = ( { navigation, flow } ) => {
 		return null;
 	}
 
-	if ( shouldShowLaunchpadFirst( site ) ) {
-		window.location.replace( `/home/${ siteSlug }` );
-		return null;
-	}
+	// if ( shouldShowLaunchpadFirst( site ) ) {
+	// 	window.location.replace( `/home/${ siteSlug }` );
+	// 	return null;
+	// }
```

Comment:
I think we should remove this code rather than comment it out.
```suggestion
```

Reply:
This was an oversight 😅